packages/ra-ui-materialui/src/form/SimpleForm.js

```diff
@@ -4,12 +4,11 @@ import { Form } from 'react-final-form';
 import arrayMutators from 'final-form-arrays';
 import { useSelector } from 'react-redux';
 import classnames from 'classnames';
-import { useTranslate } from 'ra-core';
+import { useTranslate, useInitializeFormWithRecord } from 'ra-core';
 
 import FormInput from './FormInput';
 import Toolbar from './Toolbar';
 import CardContentInner from '../layout/CardContentInner';
-import useInitializeFormWithRecord from './useInitializeFormWithRecord';
```

Comment:
if it's not used anymore, shouldn't you remove the file?

Reply:
The file has been moved to `ra-core` and is imported at L7

Reply:
See previous change in github diff